hudi-common/src/main/java/org/apache/hudi/metadata/BaseTableMetadata.java

```diff
@@ -184,10 +184,10 @@ public Option<BloomFilter> getBloomFilter(final String partitionName, final Stri
   }
 
   @Override
-  public Map<Pair<String, String>, BloomFilter> getBloomFilters(final List<Pair<String, String>> partitionNameFileNameList)
+  public Map<Pair<String, String>, BloomFilter> getBloomFilters(final List<Pair<String, String>> partitionNameFileNameList, final String metadataPartitionName)
       throws HoodieMetadataException {
-    if (!dataMetaClient.getTableConfig().isMetadataPartitionAvailable(MetadataPartitionType.BLOOM_FILTERS)) {
-      LOG.error("Metadata bloom filter index is disabled!");
+    if (!dataMetaClient.getTableConfig().getMetadataPartitions().contains(metadataPartitionName)) {
+      LOG.error("Metadata partition not found {}", metadataPartitionName);
```

Comment:
why make the error message more generic vs stating specifically. that bloom filter index is what was disabled.

Reply:
Addressed